README.md

```diff
@@ -98,7 +98,14 @@ For more complex projects, a `.env-cmdrc` file can be defined in the root direct
 ```sh
 ./node_modules/.bin/env-cmd production node index.js
 ```
+### --no-override option
 
+Sometimes you want to set env variable which is set in a file without changing env file.
```

Comment:
`Sometimes you want to set env variables from a file without overriding existing process env variables.`